### PR TITLE
[1.11] VIPs DNS is not removed even when application is deleted

### PR DESCRIPTION
**Pull request by urbanserj**
JIRA: https://jira.mesosphere.com/browse/DCOS_OSS-3982

### CODE REVIEW

**Comment by GoelDeepak** [comment]
@urbanserj The changes LGTM me. Could you please add a unit test? Also, would you know why jenkins is failing?

**Comment by urbanserj** [comment]
@GoelDeepak I've added a test case for this bug. Jenkins build is failing on windows because 1.11 branch doesn't support it.

File: apps/dcos_l4lb/src/dcos_l4lb_lashup_vip_listener.erl [hunk 1]
```diff
@@ -427,7 +425,10 @@ test_ip6(IP6, _, _) ->
     list_to_tuple(NextIP6).
 
 process_vips_tcp_test() ->
-    process_vips(tcp).
+    State = process_vips(tcp),
```
Comment (GoelDeepak) [inline apps/dcos_l4lb/src/dcos_l4lb_lashup_vip_listener.erl]: would it make sense to also assertNotEqual([], ets:tab2list(name_to_ip)) here?

Comment (urbanserj) [reply apps/dcos_l4lb/src/dcos_l4lb_lashup_vip_listener.erl]: There is an assert for that in `process_vip/1`.

https://github.com/dcos/dcos-net/pull/69/files#diff-599c9ad6aeffb3325a8c2108bcc67ed8R459

**Pull request merged**